.github/workflows/release.yaml

```diff
@@ -278,6 +280,10 @@ jobs:
       - name: Update VERSION on master branch
         run: |
           echo ${{ env.NEW_VERSION }} > VERSION
+          # Replace the 'project-release: vX.X.X-rcX' line in SECURITY-INSIGHTS.yml
+          sed -i "s/project-release: v.*$/project-release: v${{ env.NEW_VERSION }}/" SECURITY-INSIGHTS.yml
+          # Update the 'commit-hash: XXXXXXX' line in SECURITY-INSIGHTS.yml
+          sed -i "s/commit-hash: .*/commit-hash: ${{ env.NEW_VERSION }}/" SECURITY-INSIGHTS.yml
```

Comment:
So I don't think [this](https://github.com/ossf/security-insights-spec/blob/main/specification.md#header) is actually _supposed_ to be auto-updated. Since the `SECURITY-INSIGHTS.yml` is meant to be manually reviewed, it would be updated the next time it is manually reviewed, is my understanding of it

Reply:
The update will be in a PR rather than auto-committed. How about updating the last-reviewed time as well and just adding a note to the PR that the reviewer should make sure the insights doc is up to date?

Reply:
I was thinking of just manually reviewing it once a year and just updating it then. Least effort 🤷 

Ofc if we change tooling etc, then can update it simultaneously as well, but otherwise I was thinking of updating it as infrequently as possible.

Reply:
> I was thinking of just manually reviewing it once a year and just updating it then

*That* will never happen, I can assure you :sweat_smile:

Reply:
It _has to_ be manually reviewed at least once a year per the [`expiration-date`](https://github.com/ossf/security-insights-spec/blob/main/specification.md#header)

Reply:
The date has to be manually bumped every year. 😉